spread/lxd.go

```diff
@@ -19,13 +19,28 @@ import (
 )
 
 func LXD(p *Project, b *Backend, o *Options) Provider {
-	return &lxdProvider{p, b, o}
+	return &lxdProvider{
+		project: p,
+		backend: b,
+		options: o,
```

Comment:
Should this set `vm: false` (even though that's the default) just for clarity?

Reply:
done